src/wrapper/jnienv.rs

```diff
@@ -169,6 +169,24 @@ impl<'a> JNIEnv<'a> {
         )
     }
 
+    /// Tests whether an object is an instance of a class.
+    pub fn is_instance_of<T>(&self, object: JObject<'a>, class: T) -> Result<bool>
```

Comment:
Since we have a native-to-Java integration tests, I'd add at least one positive and one negative test, e.g.:
 * [String](https://docs.oracle.com/javase/9/docs/api/java/lang/String.html) is instance of CharSequence
 * String is not instance of Runnable.
 * `null` is not instance of anything.